app/src/main/java/net/kwatts/powtools/App.java

```diff
@@ -10,6 +16,10 @@
 
     public static App INSTANCE = null;
     private SharedPreferencesUtil sharedPreferencesUtil = null;
+    PowerManager.WakeLock mWakeLock;
```

Comment:
[Just Say mNo to Hungarian Notation](http://jakewharton.com/just-say-no-to-hungarian-notation/)